File: CODIGOS/codeabbey14.vb
Imports System

Public Class Program

  Public Shared Sub Main()

    Dim n As Integer = CInt(Console.ReadLine())
    Dim count As Integer = 0

    For i As Integer = 0 To n - 1
      For j As Integer = 0 To n - 1
        If i = j Then
          count = count + 1
        End If
      Next
    Next

    Console.WriteLine(count)

  End Sub

End Class

File: CODIGOS/codeabbey14.cs
using System;

public class Program {
    public static void Main(string[] args) {
        int n = int.Parse(Console.ReadLine());
        int count = 0;
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                if (i == j) {
                    count++;
                }
            }
        }
        Console.WriteLine(count);
    }
}
